packages/core/saved-objects/core-saved-objects-server/src/contracts.ts

```diff
@@ -209,4 +223,25 @@ export interface SavedObjectsServiceStart {
    * {@link SavedObjectsType | saved object types}
    */
   getTypeRegistry: () => ISavedObjectTypeRegistry;
+  /**
+   * Returns the (alias to the) index that the specified saved object type is stored in.
+   *
+   * @param type The SO type to retrieve the index/alias for.
+   */
+  getIndexForType: (type: string) => string;
+  /**
+   * Returns the (alias to the) index that the specified saved object type is stored in.
+   *
+   * @remark if multiple types are living in the same index, duplicates will be removed.
+   * @param types The SO types to retrieve the index/alias for.
+   */
+  getIndicesForTypes: (types: string[]) => string[];
```

Comment:
TSDoc was added fairly quickly, please tell me if we want to rephrase stuff in here.

Reply:
NIT plurals here would be nice:
 ```
  /**
   * Returns the (aliases to the) indices that the specified saved object types are stored in.
   *
   * @remark an index that matches multiple types will only be returned once, aka duplicates will be removed.
   * @param types The SO types to retrieve the indices/aliases for.
   */
```